AEPIdentity/Tests/IdentityFunctionalTests.swift

```diff
@@ -51,6 +51,48 @@ class IdentityFunctionalTests: XCTestCase {
         XCTAssertNotNil(sharedState?[IdentityConstants.EventDataKeys.VISITOR_ID_ECID])
     }
 
+    /// Tests that a sync event dispatches a shared state update
+    func testSyncIdentifiersForceSyncEventInstallScenario() {
```

Comment:
for these two new tests can we also check hit is queued / network request is sent?

Reply:
@emdobrin I was thinking of this but since we test directly the listeners first event will not be treated as forceSync since that is being called from isReadyForEvent. I can add a call to isReady and check for hit.